s4/clarity/lims.py

```diff
@@ -259,7 +262,7 @@ def raw_request(self, method, uri, **kwargs):
                 s4.clarity.utils.ssh.tunnel(real_host, 9080, "glsai")
                 self._opened_ssh_tunnel = True
 
-        response = self._session.request(method, uri, allow_redirects=False, **kwargs)
+        response = self._session.request(method, uri, timeout=self.timeout, allow_redirects=False, **kwargs)
```

Comment:
This is going to break a lot of code, we have a lots of clarity calls that take multiple minutes.